test/plain-list/AutosuggestApp.js

```diff
@@ -48,6 +48,13 @@ export const shouldRenderSuggestions = sinon.spy(value => {
   return value.trim().length > 0 && value[0] !== ' ';
 });
 
+export const toggleAlwaysRenderSuggestions = sinon.spy((toggle, callback) => {
+  app.setState({
+    alwaysRenderSuggestions: toggle,
+    suggestions: getMatchingLanguages(app.state.value || 'elm')
```

Comment:
For those tests to be useful it is necessary to get suggestions even when there's no value in the input – that's why I use `elm` here. It's not what would happen in a real-world app but does the same job for the tests' purpose.

I added a comment to make this clearer.